packages/material-ui/src/useScrollTrigger/useScrollTrigger.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+function getScrollY(ref = window) {
+  if (ref && ref.pageYOffset !== undefined) {
+    return ref.pageYOffset;
+  }
+  if (ref && ref.scrollTop !== undefined) {
+    return ref.scrollTop;
+  }
+  return document
+    ? (document.documentElement || document.body.parentNode || document.body).scrollTop
```

Comment:
Do you know if it's possible to simplify the `getScrollY` logic to only support browsers in this scope https://next.material-ui.com/getting-started/supported-platforms/?

Reply:
We should be fine simplifying to only [pageYOffset](https://developer.mozilla.org/en-US/docs/Web/API/Window/pageYOffset)

Reply:
Nice :)